docs/README.md

```diff
@@ -1097,6 +1097,13 @@ with the enabled de-duplication. See [this section](#deduplication) for details.
 
 VictoriaMetrics de-duplicates data points if `-dedup.minScrapeInterval` command-line flag is set to positive duration. For example, `-dedup.minScrapeInterval=60s` would de-duplicate data points on the same time series if they fall within the same discrete 60s bucket.  The earliest data point will be kept. In the case of equal timestamps, an arbitrary data point will be kept. See [this comment](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/2112#issuecomment-1032587618) for more details on how downsampling works.
 
+By default `vminsert` tries to route all the samples for a single time series to a single `vmstorage` node. In this case enabling deduplication on `vmstorage` would be enough. But samples for a single time series can be spread among multiple `vmstorage` nodes under certain conditions:
```

Comment:
```suggestion
The same value for `-dedup.minScrapeInterval` command-line flag must be passed to both `vmselect` and `vmstorage` nodes because of the following aspects.
By default `vminsert` tries to route all the samples for a single time series to a single `vmstorage` node. But samples for a single time series can be spread among multiple `vmstorage` nodes under certain conditions:
```